packages/go_router/example/lib/shell_route.dart

```diff
@@ -77,6 +77,24 @@ class ShellRouteExampleApp extends StatelessWidget {
               ),
             ],
           ),
+
+          /// The third screen to display in the bottom navigation bar.
```

Comment:
Since you are touching the example, can you write a simple test in example/test? just to make sure c/details can display the right content.